app.js

```diff
@@ -133,10 +148,10 @@ function onSearch(data) {
         dlElement.appendChild(dtElement);
         dlElement.appendChild(ddElement);
         cardBackText.appendChild(dlElement);
```

Comment:
This is the line of code that puts the dinosaur information into the cardBackText.